tests/foreman/cli/test_satellitesync.py

```diff
@@ -588,28 +597,31 @@ def test_positive_export_format_inheritance(
         :expectedresults:
             1. Incremental exports inherit the format from the previous complete export.
             2. When using --from-history-id, the format is inherited from the specified export.
+            3. Tests work with and without destination-server parameter.
 
-        :Verifies: SAT-32715, SAT-38392
+        :Verifies: SAT-32715, SAT-38691
 
-        :BlockedBy: SAT-38392
+        :BlockedBy: SAT-38691
 
         :customerscenario: true
         """
         if subject == 'library':
             cpl_export = target_sat.cli.ContentExport.completeLibrary
             inc_export = target_sat.cli.ContentExport.incrementalLibrary
             exp_params = {'organization-id': function_org.id}
-            importable_msg = f'{function_org.name}/Export-Library'
-            syncable_msg = f'{function_org.name}/Export-Library-SYNCABLE'
```

Comment:
Could we keep these vars for later assertions?

The main reason for them was to distinguish between importable and syncable (contains `SYNCABLE` or not) paths, but it also checks the export path falls under the correct organization and repo (optionally). We could extend it for the "destination" format too.
```
importable_msg = f'{function_org.name}/Export-Library'
syncable_msg = f'{function_org.name}/Export-Library-SYNCABLE'
if with_destination_server:
    destination = gen_string('alpha')
    exp_params['destination-server'] = destination
    importable_msg = f'{imporable_msg}-{destination}'
    syncable_msg = f'{syncable_msg}-{destination}'
```